### ISSUE TITLE
[Redmine Windows installer 5.0.0-1] does not install successfully on server without Internet access

### ISSUE THREAD

**Issue by jfix** [bitnami/vms#79]
### Platform

Installers

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

not available (could not upload to Bitnami server)

### bndiagnostic output

I've run the bndiagnostic tool and have the zip file. However, I have to admit I don't feel comfortable sharing the file in a public forum. I can provide the file via a private message.

Note though that the issue occurs during installation, we're not at a stage yet where running the stack is an issue, I think.

### bndiagnostic was not useful. Could you please tell us why?

I got a zip file which I can provide.

### Describe your issue as much as you can

My issue has been described very well in this existing ticket on the system that will be switched off on 15 July (in 10 days' time):

https://community.bitnami.com/t/about-installing-a-server-that-is-not-connected-to-the-internet-bitnami-redmine-4-4-2-0-windows-x64-installer-exe/99391

Basically, we want to install Redmine on an internal server without access to the Internet, but apparently, the installer attempts to download a msys2 dependency as you can see from the following screenshot.

![image](https://user-images.githubusercontent.com/170240/177342367-914f829e-0d04-4d78-a812-78ca35ab76ea.png)

And the failure of downloading the tool leads to an unsuccessful installation. I tried to install msys2 manually, but it seems that this didn't help.

Grateful for your help.

**Comment by mdhont** [comment]
Can you try the latest version of Redmine for Windows?

https://bitnami.com/redirect/to/2156927/bitnami-redmine-5.0.0-2-windows-x64-installer.exe

**Comment by jfix** [comment]
Things went much more smoothly with this version! 👍 

However, the process continues until the installation of the Windows services for MariaDB and Apache. In the screenshot below, you'll see that the services are installed and running, but the install screen does not proceed to the next step (the end I'd assume). It is stuck at this stage for about 5 minutes. 

I guess there is still the thin server Windows service to be installed ...

![image](https://user-images.githubusercontent.com/170240/177732205-5b653329-cc73-4460-ae83-ca38cba2ed9f.png)

Thank you for your support so far!

**Comment by jfix** [comment]
Sorry, forget my previous message, the process has finally continued after a while, it is now initializing the database. I'll give it some more time. 👍

**Comment by jfix** [comment]
Everything seems ok! 👍 Thank you!

![image](https://user-images.githubusercontent.com/170240/177734128-2f647fc6-0527-4b2d-ac41-828ea3f15521.png)
